File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.js
/**
 * @param {string[]} a
 * @return {number}
 */
var finalValueAfterOperations = function(a) {

 let i,k;
k=0;
i=0;
    for(i=0;i<a.length;i++)
    {
       if(a[i]==="X++" || a[i]==="++X")
       {
           k+=1;
       }
     else if(a[i]==="X--" || a[i]==="--X")
     {
            k-=1;
     }
    }
         return k;
    
};

File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.c
int finalValueAfterOperations(char** a, int n) {


 int k,i;

k=0;
 for(i=0;i<n;i++)
 {
   // if(strcmp(a[i]=="--x")==0 || strcmp(a[i]=="x--")==0)
   if(strcmp(a[i], "--X") == 0 || strcmp(a[i], "X--") == 0)
    {
        k=k-1;
    }
  // else if(strcmp(a[i]=="x++")==0 || strcmp(a[i]=="++x")==0)
  else if(strcmp(a[i], "++X") == 0 || strcmp(a[i], "X++") == 0)
    {
        k=k+1;
    }
 }

   return k;

}